### PR TITLE
Popup modal

### PR DESCRIPTION
**Pull request by Alusp**
### In this PR

-  Added A popup modal Feature
- Fix the linters errors as requested by the previous reviewer

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -140,4 +140,13 @@ function projectPopup(position) {
...
+const projectnBtn = document.querySelectorAll('.see-project');
+projectnBtn.forEach((btn, ind) => {
+  btn.addEventListener('click', () => {
+    projectPopup(ind);
+    const popup = document.querySelector('.popup');
+    document.querySelector('.close-icon').addEventListener('click', () => {
+      portfolios.removeChild(popup);
+    });
+  });
+});
```
Comment (cynthiainga) [inline script.js:152]: - [ ] Good implementation of popup, but you still have something to fix on your design. The popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) :upside_down_face: :upside_down_face:. In your case when I open the popup to see the project content I need to scroll down, I think it should be better when the user clicks the `see project` button can see all the popup content without scrolling down :grin: :grin: 

Your design
Screenshot from 2022-07-21 19-22-09

Figma design
Screenshot from 2022-07-21 19-05-21

- [ ] Please kindly make sure your image is visible, so then the user can see very well the project's picture :grin: :grin: :upside_down_face:

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors :heavy_check_mark: 
- [ ] Good commit history :heavy_check_mark: 
- [ ] Correct GitHub Flow :heavy_check_mark: 

## Required Changes ♻️

- [ ] Good implementation of your projects array, but I think it should be better if you can add for each project in your array the `link_to_live_demo` and `link_to_source_code`, so then in the future, it will be easy for you to add a project to your portfolio :upside_down_face: :upside_down_face:
example:
```javascript
const projects = [{
    name: "Tonic",
    description: "Lorem Ipsum....",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    liveLink: "link_to_live_version.com",
    sourceCode: "link_to_githu_repo",
}]
```
- [ ] Please kindly check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ OPTIONAL] I noticed that your closed button for the mobile version is visible even in the desktop version, is supposed to be invisible in the desktop version if you don't mind kindly if that.
Screenshot from 2022-07-21 19-45-35

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (codecaiine) [reply script.js:176]: - [ ] [REQUIRED]  Good job for implementing the changes requested by the previous reviewer, your popup looks very nice :clap:  but I think you can improve your popup by increasing the size of your popup. Please make sure that you have the dimensions as on the Figma template and increase your popup background color  `background: rgb(193 199 208 / 50%).`  and   ` backdrop-filter: blur(3px)`  to cover the all website details behind your popup :+1: 

- [ ] [REQUIRED] The previous reviewer requested changes to your project pictures to make it more visible and more clear for use please apply the changes :smile: 

#### Your project
Screenshot from 2022-07-22 00-06-18

#### Figma
Screenshot from 2022-07-22 00-24-07

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @Team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Descriptive Pull request :heavy_check_mark: 
- Good use of Gitflow :heavy_check_mark: 
- Commits are meaningful :heavy_check_mark: 
- Clean code :heavy_check_mark:  
- Great UI :heavy_check_mark:

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @Alusp 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**